protocol/avdl/keybase1/gui_file_access.avdl

```diff
@@ -0,0 +1,160 @@
+@namespace("keybase.1")
+
+protocol GuiFileAccess {
+
+  import idl "common.avdl";
+
+/*
+  File manipulation interface
+  
+  Caller provides OpID to asynchronous method call, then polls 
+  by calling a get* method until status is no
+  longer pending or operation is cancelled.
+
+  Caller can optionally block by calling wait
+ 
+  KBFS paths are expected to start with "private" or "public" - mount name is implied.
+  Other paths are absolute, full paths.
+*/
+
+  // Random GUID provided by caller 
+  fixed OpID(16);
+  
+  enum ItemType {
+    FILE,
+    DIR,
+    SYM,
+    EXEC
+  }
+
+  record Item {
+    string path;
+    Time time;
+    int size;   // size of file or count of dir entries (TBD - mmay be expensive/unnecessary)
+    ItemType itemType;
+  }
+
+  @typedef("int")
+  record ErrornNum {}
+
+  enum Status {
+    finished,
```

Comment:
Use the style we use elsewhere for these guys, including pinning the values with `_1`, etc.